docs/source/tutorials/building_blocks.rst

```diff
@@ -41,7 +41,7 @@ the time, you will be dealing with vmobjects, though we will continue to use
 the term "mobject" to refer to the class of shapes that can be displayed on
 screen, as it is more general.
 
-.. note:: Any object that can be displayed on screen is a ``mobject``, even if
+.. note:: Any object that can be displayed on the screen is a ``mobject``, even if
```

Comment:
see prev. comment.

Reply:
Same

Reply:
it kind of depends on whether you are using the idiom "on screen" or whether you are thinking of a "screen" as being an abstract canvas within manim in which we display objects.

In the end both choices are fine.